pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -68,6 +68,11 @@ in rustPlatform.buildRustPackage {
     bash ./script/build-wasm --debug
   '';
 
+  postInstall = ''
+    export PREFIX=$out
+    make install
```

Comment:
that looks wrong. make install should be called by default. If you need to export PREFIX, it can be done via configureFlags.

Reply:
nevermind it uses buildRustPackage.

Reply:
this seems to install both static and shared libary. we should pick one. I think you (or another) could propose a patch for their Makefile to be more granular. Since homebrew is packaging it too, would be interesting to see how they've done it.

Reply:
In the arch community repo they package the shared library https://www.archlinux.org/packages/community/x86_64/tree-sitter/files/

Reply:
I went to the liberty of addressing this in a follow-up commit; thanks for your contribution @colemickens!

@teto can you re-review and then I can merge?